ui/src/app/views/concept-table/component.tsx

```diff
@@ -0,0 +1,140 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Clickable} from 'app/components/buttons';
+import {CheckBox} from 'app/components/inputs';
+import {PopupTrigger} from 'app/components/popups';
+import {ReactWrapperBase, toggleIncludes} from 'app/utils';
+import {reactStyles} from 'app/utils';
+import {Concept} from 'generated/fetch/api';
+import * as fp from 'lodash/fp';
+import {Column} from 'primereact/column';
+import {DataTable} from 'primereact/datatable';
+import * as React from 'react';
+
+const styles = reactStyles({
+  colStyle: {
+    lineHeight: '0.5rem',
+    textAlign: 'center'
+  }
+});
+
+interface ConceptTableProps {
+  concepts: Concept[];
+  loading: boolean;
+  onSelectedChanged: Function;
+}
+
+interface ConceptTableState {
```

Comment:
nit: i think we've been defining these inline in the class declaration rather than in their own `interface` blocks to reduce boilerplate